scripts/build_utils.sh

```diff
@@ -1287,3 +1287,12 @@ EOF
                  ${build_area}/SPECS/ceph-release.spec
     fi
 }
+
+# dockerhub started aggressively rate limiting in November 2020. We can use an internal docker mirror if the builder is in the upstream lab.
+use_internal_container_registry() {
+  if ping -c 1 docker-mirror.front.sepia.ceph.com; then
```

Comment:
It might be more reliable and a better test of an available service to actually http-transfer something, like with curl, and maybe a timeout.  I think ping can fail for silly reasons sometimes.

Reply:
Good point.  Updated.